sdk/core/azure-core/README.md

```diff
@@ -110,6 +109,8 @@ The Configuration object is the home of all the configurable policies in the pip
 A new Configuration object provides *no default policies*.
 It is up to the SDK developer to specify each of the policy defaults as required by the service.
 
+*Configuration should not be exposed part of the public API of the resulting SDK.*
```

Comment:
Seems to be missing a word. "not be an exposed part of" or "not be exposed as part of" ?

Reply:
lol for me it was correct :p. I guess the right one is "not be exposed as part of". Excuse my english :)

Reply:
fixed